integration-tests/src/main/java/io/druid/testing/ConfigFileConfigProvider.java

```diff
@@ -62,20 +63,26 @@ private void loadProperties(String configFile)
     catch (IOException ex) {
       throw new RuntimeException(ex);
     }
-    routerHost = props.get("router_host") + ":" + props.get("router_port");
+    // there might not be a router; we want routerHost to be null in that case
+    routerHost = props.get("router_host");
+    if (null != routerHost) {
+	routerHost += ":" + props.get("router_port");
```

Comment:
would be better if this can use HostAndPort. But since this PR keeps with prior convention, I do not consider it a blocker.